menuinst/platforms/base.py

```diff
@@ -43,13 +45,13 @@ def __init__(
         else:
             self.env_name = self.prefix.name
 
-    def create(self) -> List[Path]:
+    def create(self) -> Iterable[os.PathLike]:
```

Comment:
```suggestion
    def create(self) -> tuple[os.PathLike]:
```

The functions for all platforms return `tuple`, but since this function actually doesn't return anything, this is just for consistency.